docker-compose.yml

```diff
@@ -1,75 +1,68 @@
+# An demo version of WordPress for easy scaling
 wordpress:
-  build: .
-  privileged: true
-  env_file: _env
-  environment:
-    - CONTAINERBUDDY=file:///etc/containerbuddy.json
+  image: autopilotpattern/wordpress:latest
```

Comment:
We'll need some image here so we can demo this easily. It will just have the default themes, but that's enough.